src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/caseload/ApproverCaseloadServiceTest.kt

```diff
@@ -24,4 +45,151 @@ class ApproverCaseloadServiceTest {
     service.getApprovalNeeded(aListOfPrisonCodes)
     verify(prisonApproverService, times(1)).getLicencesForApproval(listOf("ABC", "DEF"))
   }
+
+  @Test
+  fun `It builds the approval needed caseload`() {
```

Comment:
Will be adding more tests in another PR but this matches the unit test on the front end currently.